proc_IR.py

```diff
@@ -7,10 +7,11 @@
 save_npz = False
 #}}}
 coherence_pathway = {'ph1':0,'ph2':1}
+# this should not be part of the PR
 for thisfile,exp_type,nodename,postproc,f_range,t_range,clock_correction,IR,ILT in [
-       ('210610_water_TempCont_probe_DNP_1','ODNP_NMR_comp/ODNP',
-           'FIR_34dBm','spincore_IR_v1',
-           (0.1e3,0.345e3),(None,50e-3),True,False,False),
+       ('210614_TEMPOL_500uM_cap_probe_DNP','ODNP_NMR_comp/test_equipment',
+           'FIR_30dBm','spincore_IR_v1',
+           (-0.08e3,0.15e3),(None,60e-3),False,False,False),
```

Comment:
changes to this file shouldn't be part of the pull request!

Reply:
using data I know is good to test the correlation alignment in an actual experiment. The temp control stuff at this point was still sketchy so I'm using data I know is good